### PR TITLE
Modify RHUI_CLOUD_MAP as per arch + add some comments

### PR DESCRIPTION
**Pull request by Rezney**
We found out that on RHEL-ALT ARM arch the el7 RHUI user rpm package have different name.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/checkrhui/actor.py [hunk 1]
```diff
@@ -32,7 +34,8 @@ class CheckRHUI(Actor):
     tags = (ChecksPhaseTag, IPUWorkflowTag)
 
     def process(self):
-        for provider, info in rhui.RHUI_CLOUD_MAP.items():
+        arch = platform.machine()
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkrhui/actor.py:37]:
```suggestion
        arch = self.configuration.architecture

File: repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py [hunk 2]
```diff
@@ -319,11 +320,13 @@ def _get_rh_available_repoids(context, indata):
     RHUI special packages (every cloud provider has itw own rpm).
     """
 
+    arch = platform.machine()
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py:323]:
```suggestion
    arch = api.current_actor().configuration.architecture
```

**Review by pirat89 — CHANGES_REQUESTED** [review]
Do not use `platform` anywhere. Take the information about the architecture from the actor configuration.
```
self.configuration.architecture

# or
api.current_actor().configuration.architecture
```

**Comment by Rezney** [comment]
e2e tests

**Comment by Rezney** [comment]
e2e tests

**Comment by RadekBiba** [comment]
Upgrading ARM64 and x86_64 VMs works for me with leapp-repository-0.11.0-0.202009170724Z.e007094.rhui_map_per_arch.el7.noarch in AWS (using RHUI).

**Review by pirat89 — APPROVED** [review]
lgtm. @Rezney  let's celebrate this big milestone  :D

**Pull request merged**